.github/workflows/tests-smoke.yaml

```
@@ -171,15 +179,22 @@ jobs:
       - name: Capture cilium-sysdump
         if: ${{ failure() }}
         run: |
+          curl -sLO https://github.com/cilium/hubble/releases/download/v0.8.1/hubble-linux-amd64.tar.gz
```

Should this use latest so we always pick up the stable release?

```suggestion
          curl -sLO https://github.com/cilium/hubble/releases/download/latest/hubble-linux-amd64.tar.gz
```

---

Done